cmd/gmnhg/templates.go

```diff
@@ -35,20 +35,22 @@ func defineFuncMap() template.FuncMap {
 	return fm
 }
 
-var defaultSingleTemplate = mustParseTmpl("single", `# {{ .Metadata.Title }}
-
+var defaultSingleTemplate = mustParseTmpl("single", `# {{ or .Metadata.Title (trimPrefix "/" .Link) }}
```

Comment:
Is a plain URI any better than no title?

Reply:
Good point, I will just cut out the title entirely if one isn't provided.